extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+# 3DTILES_implicit_tiling Extension
```

Comment:
Like https://github.com/CesiumGS/glTF/pull/1, all sample tilesets that we send from here on out should have an `extras.draftVersion` that follows semver and distinguishes it from future revisions.

```
"extensions": {
  "3DTILES_implicit_tiling": {
    "extras": {
      "draftVersion": "0.0.0"
    },
    ...
  }
}
```

Reply:
Just to clarify, no need to edit the examples in the spec, just make sure to include in sample tilesets we send out.